reana_client/api/client.py

```diff
@@ -166,7 +202,16 @@ def get_workflow_status(workflow, access_token):
 
 
 def create_workflow(reana_specification, name, access_token):
-    """Create a workflow."""
+    """Create a workflow.
+
+    :param reana_specification: a dictionary representing the REANA specification of the workflow.
+    :param name: name of the workflow.
+    :param access_token: access token of the current user.
+
+    :return: if the workflow was created successfully, a dictionary with the information about
+             the ``workflow_id`` and ``workflow_name``, along with a ``message`` of success.
+             Otherwise, only a ``message`` key is returned, with the error message.
```

Comment:
I think the error message is not returned but an exception is raised, am I wrong? If so, let's just remove this phrase.